packages/slice-machine/package.json

```diff
@@ -38,9 +38,11 @@
   },
   "dependencies": {
     "@babel/register": "^7.11.5",
+    "@prismicio/api-renderer": "^0.1.1",
+    "@prismicio/mocks": "^0.1.1",
     "@prismicio/slice-simulator-com": "^0.2.0",
-    "@prismicio/types-internal": "^0.2.7",
-    "@faker-js/faker": "^6.0.0",
+    "@prismicio/types-internal": "^0.2.8",
+    "@reach/menu-button": "^0.16.2",
```

Comment:
Weird it says you added this dependency, it's suppose to be already there.

Reply:
idk